apps/site/src/lib/components/ValidatingForm/ValidatingForm.module.scss

```diff
@@ -0,0 +1,9 @@
+.validated {
+	&:invalid p {
+		display: block;
+	}
+}
+
+.notYetValidated p {
+	display: none;
+}
```

Comment:
Nitpick [non-blocking]: bit weary of selecting on `p` elements instead of a more specific class name, but this will work for now. Coincidentally, this will already work out for the Guest Login page.

Reply:
Implemented in [`bc51652`](https://github.com/HackAtUCI/irvinehacks-site-2024/pull/114/commits/bc516527b7a991b47d5d1b2bb1863191b61a020b).

Reply:
Okay, note for later that we'll need to make the corresponding change on the Guest Login `VerificationForm`.